functions/listeners/afaf/afaf.js

```diff
@@ -1,12 +1,11 @@
+const { MessageEmbed } = require('discord.js');
 const { blaccLogo, blaccSmithServer } = require('../../constants');
-const { discordGuilds, discordChannels } = require('../config');
 
-exports.handleAFAF = async (message) => {
+exports.handleAFAF = async ({discordClient, discordGuilds, message}) => {
```

Comment:
The only thing that needs to be passed here is the message. Import discordClient and discordGuilds from the listeners config